packages/eas-cli/src/utils/git.ts

```diff
@@ -56,6 +56,12 @@ async function getBranchNameAsync(): Promise<string | undefined> {
   } catch (e) {}
 }
 
+async function getLastCommitMessageAsync(): Promise<string | undefined> {
+  try {
+    return (await spawnAsync('git', ['--no-pager', 'log', '-1', '--pretty=%B'])).stdout;
+  } catch (e) {}
```

Comment:
```suggestion
async function getLastCommitMessageAsync(): Promise<string | null> {
  try {
    return (await spawnAsync('git', ['--no-pager', 'log', '-1', '--pretty=%B'])).stdout.trim();
  } catch (e) {
    return null;
  }
```
Explicit is better than implicit.

Also added `.trim()` to remove whitespace from the output.